packages/components/src/styles/table-cell/table-cell.stories.ts

```diff
@@ -41,580 +41,213 @@ export const Default = {
-
-            sortData[index] = 'none';
-            icons[index].setAttribute('name', iconName);
-            headerCell.removeAttribute('aria-sort');
-          }
-        });
-      }
-    };
-
-    const sortTableByColumn = (table: HTMLTableElement | null, column: number, descending: boolean) => {
-      if (table) {
-        const dirModifier = descending ? 1 : -1;
-        const tableBody = table.tBodies[0];
-        const rows = Array.from(tableBody.querySelectorAll('tr'));
-
-        // Sort each row
-        const sortedRows = rows.sort((a, b) => {
-          const firstCol = a.querySelector(`td:nth-child(${column + 1})`)?.textContent?.trim();
-          const secondCol = b.querySelector(`td:nth-child(${column + 1})`)?.textContent?.trim();
-
-          if (firstCol && secondCol) {
-            return firstCol > secondCol ? dirModifier : -1 * dirModifier;
-          }
-
-          return 0;
-        });
-
-        // Remove all existing TRs from the table
-        while (tableBody.firstChild) {
-          tableBody.removeChild(tableBody.firstChild);
-        }
-
-        // Re-add the newly sorted rows
-        tableBody.append(...sortedRows);
-      }
-    };
 
+export const Variants = {
+  render: () => {
     return html`
-      <style>
-        .story-wrapper {
-          display: inline-block;
-          max-width: 1200px;
-        }
-
-        .headline {
-          padding: 16px;
-          background: #e0e0e0;
-          text-align: left;
-          font-size: 14px;
-          font-weight: bold;
-          width: 100%;
-          box-sizing: border-box;
-        }
-
-        .disclaimer {
-          background-color: #f6f6f6;
-          padding: 16px;
-          text-align: left;
-          font-size: 14px;
-          color: #333;
-          width: 100%;
-          margin-bottom: 10px;
-          box-sizing: border-box;
-        }
-
-        .sample-table {
-          margin-top: 20px;
-          width: 100%;
-        }
-
-        .sd-table-cell.sortable {
-          padding: 0;
-        }
-
-        .sd-table-cell.sortable button {
-          display: inline-flex;
-          box-sizing: border-box;
-          height: 100%;
-          width: 100%;
-          margin: 0;
-          padding: 16px;
-          align-items: center;
-          justify-content: flex-start;
-          gap: 8px;
-          text-align: left;
-        }
-
-        div:not(:first-of-type).headline {
-          margin-top: 72px;
-        }
-      </style>
-      <div class="story-wrapper">
-        <div class="headline">Simple Table</div>
-        <table class="sd-table sample-table">
-          <thead>
-            ${(() => {
-              return html`<tr>
-                ${headerData.map(cellData => {
-                  return html`<th class="sd-table-cell sd-table-cell--bg-white">${cellData}</th>`;
-                })}
-              </tr>`;
-            })()}
-          </thead>
+      <div class="flex gap-4">
+        <table class="sd-table">
```

Comment:
IMO it's probably better to have the different variants displayed in a single table, what do you think?

Reply:
Updated.